docs/docs/guides/airy-core-in-test-env.md

```diff
@@ -129,8 +130,8 @@ data:
     trust_host_root_certs: true
 ```
 
-If you prefer to use your own Ngrok implementation or point the Ngrok client to
-connect to the service provided by the Ngrok company at `https://ngrok.io`,
+If you prefer to use your own ngrok implementation or point the ngrok client to
+connect to the service provided by the ngrok company at `https://ngrok.io`,
```

Comment:
What's the reason we are lowercasing "Ngrok"? 🤔  Isn't it a special name like "Facebook", "Google" or "Rasa"?

Reply:
generally I go for the casing of the official docs and as far as I can ngrok is all down case

Reply:
Ah, that's an even better rule 👍